src/de/dkfz/b080/co/aceseq/ACESeqWorkflow.java

```diff
@@ -81,10 +82,13 @@ public boolean execute(ExecutionContext context, BasicBamFile _bamControlMerged,
             mergedSvs = ACESeqMethods.mergeSv(breakpoints.value0, runWithSv); // true is passed
```

Comment:
As we sometimes start "ACEseq part I" (until SV file is required) in parallel to SV calling. The pipeline should still submit all Jobs until this point. A message should appear to the user (e.g. "No SV file found, only the first part of the ACEseq workflow will be run") and the second part should not be executed.

Reply:
What do you mean here? The code here has not changed, but of course we can arrange this. I'm only not sure, under which condition the message should be issued. In the runWithCrest branch, right?